public/scripts/worktrollyUI.js

```diff
@@ -22,6 +22,25 @@ $("#logWorkPage").click(function() {
     fillDataIntoLogWorkPage(id, title, estimatedTime, logWorkTotalTime, workDone, sprintNumber);
 });
 
+$("#editPage").click(function() {
+    var id = $("#idTaskDescription").html();
+    var title = $("#titleTaskDescription").html();
+    var des = $("#descriptionTaskDescription").html();
+    var priority = $("#priorityTaskDescription").html();
+    var difficulty = $("#difficultyTaskDescription").html();
+    var creator = $("#creatorTaskDescription").html();
+    var assignee = $("#assigneeTaskDescription").html();
+    var estimatedTime = $("#estimatedTimeTaskDescription").html();
+    var status = $("#statusTaskDescription").html();
+    var category = $("#categoryTaskDescription").html();
+    // var storyPointNumber = $("#storyPointEditPageTask").html();
+    var sprintNumber = $("#sprintNumberTaskDescription").html();
+    console.log(title);
```

Comment:
remove console.log(title);
it was for checking purpose

Reply:
removed